### PR TITLE
bump helm-operator-plugins version

### LINKED ISSUE #5771: Cut release for helm-operator-plugins and bump version in sdk
there was a fix in helm-operator-plugins to bump the golang version in the scaffolding [here](https://github.com/operator-framework/helm-operator-plugins/commit/e484968a09c1eca03eaadb418449c33bef3579c4). Currently if you try to build a hybrid-helm plugin it blows up with a cryptic error. Could we cut a new release of helm-operator-plugins and bump the version in sdk?

### PR DESCRIPTION
**Pull request by everettraven**
Signed-off-by: Bryce Palmer <bpalmer@redhat.com>

**Description of the change:**
Bump helm-operator-plugins to v0.0.10

**Motivation for the change:**
resolves #5771 

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Review by varshaprasad96 — APPROVED** [review]
/lgtm

**Review by fabianvf — APPROVED** [review]

File: changelog/fragments/bump-helm-operator-plugins.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - description: >
+      Bump the helm-operator-plugins dependency version to v0.0.10.
+
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "change"
```
Comment (fabianvf) [inline changelog/fragments/bump-helm-operator-plugins.yaml:13]: is this getting rolled into description or is just highlighted weird here?

**Review by rashmigottipati — APPROVED** [review]
/lgtm

File: changelog/fragments/bump-helm-operator-plugins.yaml (hunk 1 above)
Comment (everettraven) [reply changelog/fragments/bump-helm-operator-plugins.yaml:13]: From what I can tell locally it matches the template file so this seems like it isn't getting rolled into the description, so I think it is likely just a weird highlight

**Pull request merged**